TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+[knative-dev mailing list](https://groups.google.com/forum/#!forum/knative-dev)
+(note: these are accessible to members of the knative-users list as well).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as
+[issues in the community repo](https://github.com/knative/community/issues), and
+the trademark committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee Members
+
+| Member                   | Organization | Profile |
+| ------------------------ | ------------ | ------- |
+| Unknown - to be selected | Google       | Unknown |
+| Unknown - to be selected | IBM/RedHat   | Unknown |
+| Unknown - to be selected | VMware       | Unknown |
+
+## Decision process
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding assignment of the mark, charter,
+conformance, and scope.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it. Vetoes by
+absent committee members must be presented within 5 business days of a published
+decision.
+
+Quorum is considered reached when at least half of the members are present.
+Quorum may be established over email or electronic chat systems as well as via
+video or in-person meetings.
+
+In case of extended absence or removal from the project for a code of conduct
+violation, the organization of the absent member may appoint a single
+replacement delegate from the same company during the absence.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the steering committee at
+   [knative-trademark@googlegroups.com](mailto:knative-trademark@googlegroups.com).
+   This is a private discussion list to which all members of the committee have
+   access.
+2. Open an issue on a knative repository and indicate that you would like
+   attention from the trademark committee.
+
+## Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year, as well as
+the trademark holder. These representatives shall be confirmed by the Knative
+Steering Committee; companies which have a currently-unfilled seat on the
+Trademark Committee may veto decisions made while the seat was empty as if those
+decisions had been made at the time of confirmation.
+
+Company representatives serving on the Trademark Committee may individually
+resign or be removed from their seats by their affiliated company with at least
+5 days notice.
+
+For the 2020 year, one seat is assigned to each of Google, IBM/RedHat, and
+VMware. The committee shall review this composition and the selection rules at
+least annually in October of each year (to be aligned with the Steering
+elections process).
```

Comment:
s/ (to be aligned with the Steering elections process) // -- this is covered in the first sentence of the next section.

Reply:
Cleared out the entire sentence, which had been put together independently of a section written by @pmorie